File: ruby/lib/sms.rb
class SMS
  @endpoint = "http://center.fibosms.com/service.asmx"
  @clientNo = ""
  @clientPass = ""

  def initialize(clientNo, clientPass)
    @clientNo = clientNo
    @clientPass = clientPass
  end

  def send_sms(message)
    return false
  end
end

File: python/lib/sms.py
class SMS:
  endpoint = "http://center.fibosms.com/service.asmx"
  clientNo = ""
  clientPass = ""

  def __init__(self, clientNo, clientPass):
    self.clientNo = clientNo
    self.clientPass = clientPass

  def send_sms(self, message):
    return False

